widget/playground/src/components/TokensPanel/TokensPanel.Chip.tsx

```diff
@@ -0,0 +1,36 @@
+import type { BlockchainProps } from './TokensPanel.types';
+
+import { Typography } from '@rango-dev/ui';
+import React from 'react';
+
+import { ContainerChip } from './TokensPanel.styles';
+
+export const BlockchainChip = (props: BlockchainProps) => {
+  const { label, itemCountLabel, isSelected, onClick } = props;
+  return (
```

Comment:
[Screencast from 23-11-01 15:29:27.webm](https://github.com/rango-exchange/rango-client/assets/120931936/62242c12-ca9c-4f83-8211-b0a2d091f009)

I think it's better after changing the Supported Tokens, if the selected token was the same token that we deselect, the default value of the token and the token that was selected in the embedded should be reset.

Reply:
fixed.